<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentitySettings</class>
 <widget class="QWidget" name="IdentitySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>607</width>
        <height>639</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,1">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="publicGroup">
         <property name="title">
          <string>Public Information</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="publicFieldsLayout">
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="userNameLabel">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="userName">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="statusMessageLabel">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="statusMessage">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="toxGroup">
         <property name="title">
          <string>Tox ID</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="CroppingLabel" name="toxIdLabel">
            <property name="toolTip">
             <string comment="Tox ID tooltip">This bunch of characters tells other Tox clients how to contact you.
Share it with your friends to communicate.</string>
            </property>
            <property name="text">
             <string>Your Tox ID (click to copy)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toxId">
            <property name="toolTip">
             <string comment="Tox ID tooltip">This bunch of characters tells other Tox clients how to contact you.
Share it with your friends to communicate.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="verticalFrame">
            <layout class="QHBoxLayout" name="qrGroup">
             <item>
              <widget class="QLabel" name="qrCode">
               <property name="text">
                <string>QRCODE</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="horizontalFrame">
               <layout class="QVBoxLayout" name="qrButtons">
                <item alignment="Qt::AlignTop">
                 <widget class="QLabel" name="qrLabel">
                  <property name="text">
                   <string>This QR code contains your Tox ID. You may share this with your friends as well.</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignVCenter">
                 <widget class="QPushButton" name="saveQr">
                  <property name="text">
                   <string>Save image</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignVCenter">
                 <widget class="QPushButton" name="copyQr">
                  <property name="text">
                   <string>Copy image</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="profilesGroup">
         <property name="title">
          <string>Profiles</string>
         </property>
         <layout class="QVBoxLayout" name="profilesVLayout">
          <item>
           <layout class="QHBoxLayout" name="profilesComboLayout">
            <item>
             <widget class="QLabel" name="profilesLabel">
              <property name="text">
               <string>Available profiles:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="profiles">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string comment="toolTip for currently set profile">Currently selected profile.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="profilesButtonsLayout">
            <item>
             <widget class="QPushButton" name="loadButton">
              <property name="toolTip">
               <string comment="tooltip for loading profile button">Load selected profile and switch to it.</string>
              </property>
              <property name="text">
               <string comment="load profile button">Load</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="renameButton">
              <property name="toolTip">
               <string comment="tooltip for renaming profile button">Rename selected profile.</string>
              </property>
              <property name="text">
               <string comment="rename profile button">Rename</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportButton">
              <property name="toolTip">
               <string comment="tooltip for profile exporting button">Allows you to export your Tox profile to a file.
Profile does not contain your history.</string>
              </property>
              <property name="text">
               <string comment="export profile button">Export</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="toolTip">
               <string comment="delete profile button tooltip">Delete selected profile.</string>
              </property>
              <property name="text">
               <string comment="delete profile button">Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="profilesButtonsLayout2">
            <item>
             <widget class="QPushButton" name="importButton">
              <property name="toolTip">
               <string comment="tooltip for importing profile button">Import Tox profile from a .tox file.</string>
              </property>
              <property name="text">
               <string comment="import profile button">Import a profile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newButton">
              <property name="toolTip">
               <string comment="tooltip for creating new Tox ID button">Create new Tox ID and switch to it.</string>
              </property>
              <property name="text">
               <string comment="new profile button">New Tox ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CroppingLabel</class>
   <extends>QLabel</extends>
   <header location="global">src/widget/croppinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
